app/theme.css

```diff
@@ -22,6 +23,7 @@
     --sky: #EBF7FF;
     --azure: #4095F7;
     --sapphire: #0E73F6;
+    --sapphire-filter: invert(33%) sepia(35%) saturate(7491%) hue-rotate(206deg) brightness(102%) contrast(93%);
```

Comment:
Il existe deja ligne 74, par contre les % ne sont pas les memes 🤔

Reply:
ah oui bien vu, j'avais noté que je devais reprendre l'autre après les rebase mais j'ai oublié ^^' merci du rappel